"stack split at bad time" error running tests on linux/i386 with Go1.15

```
--- FAIL: TestBuild (6.82s)
    dlv_test.go:101: 2020-08-14T16:48:38Z debug layer=rpc API server pid = 2349
    dlv_test.go:104: 2020-08-14T16:48:38Z info layer=debugger launching process with args: [/delve/_fixtures/buildtest/__debug_bin]
    dlv_test.go:104: 2020-08-14T16:48:38Z debug layer=rpc <- RPCServer.SetApiVersion(api.SetAPIVersionIn{"APIVersion":2})
    dlv_test.go:104: 2020-08-14T16:48:38Z debug layer=rpc -> *api.SetAPIVersionOut{} error: ""
    dlv_test.go:104: 2020-08-14T16:48:38Z debug layer=rpc (async 2) <- RPCServer.Command(api.DebuggerCommand{"name":"continue","ReturnInfoLoadConfig":null})
    dlv_test.go:104: 2020-08-14T16:48:38Z debug layer=debugger continuing
    dlv_test.go:104: runtime: newstack at runtime.acquireLockRank+0x18 sp=0xa090dac stack=[0xa090000, 0xa091000]
    dlv_test.go:104: 	morebuf={pc:0x807cd38 sp:0xa090db0 lr:0x0}
    dlv_test.go:104: 	sched={pc:0x8052268 sp:0xa090dac lr:0x0 ctxt:0x0}
    dlv_test.go:104: runtime.casgstatus(0xa0000e0, 0x2, 0x3)
    dlv_test.go:104: 	/usr/local/go/src/runtime/proc.go:817 +0x58 fp=0xa090de0 sp=0xa090db0 pc=0x807cd38
    dlv_test.go:104: runtime.reentersyscall(0x80d80c5, 0xa090e0c)
    dlv_test.go:104: 	/usr/local/go/src/runtime/proc.go:3078 +0x66 fp=0xa090e00 sp=0xa090de0 pc=0x8081b36
    dlv_test.go:104: runtime.entersyscall()
    dlv_test.go:104: 	/usr/local/go/src/runtime/proc.go:3127 +0x17 fp=0xa090e0c sp=0xa090e00 pc=0x80a29f7
    dlv_test.go:104: syscall.Syscall6(0x131, 0xffffff9c, 0xa0a6000, 0xa0a4000, 0x80, 0x0, 0x0, 0xa0a4000, 0xe7ca010c, 0x0)
    dlv_test.go:104: 	/usr/local/go/src/syscall/asm_linux_386.s:44 +0x5 fp=0xa090e10 sp=0xa090e0c pc=0x80d80c5
    dlv_test.go:104: syscall.readlinkat(0xffffff9c, 0x8101c55, 0xe, 0xa0a4000, 0x80, 0x80, 0x0, 0x0, 0x0)
    dlv_test.go:104: 	/usr/local/go/src/syscall/zsyscall_linux_386.go:90 +0x123 fp=0xa090e6c sp=0xa090e10 pc=0x80d7b93
    dlv_test.go:104: syscall.Readlink(0x8101c55, 0xe, 0xa0a4000, 0x80, 0x80, 0x0, 0x0, 0x0)
    dlv_test.go:104: 	/usr/local/go/src/syscall/syscall_linux.go:164 +0x75 fp=0xa090eac sp=0xa090e6c pc=0x80d7615
    dlv_test.go:104: os.Readlink(0x8101c55, 0xe, 0x0, 0x0, 0x0, 0x0)
    dlv_test.go:104: 	/usr/local/go/src/os/file_unix.go:368 +0xba fp=0xa090f18 sp=0xa090eac pc=0x80db22a
    dlv_test.go:104: os.glob..func1(0x0, 0x0, 0x0, 0x0)
    dlv_test.go:104: 	/usr/local/go/src/os/executable_procfs.go:29 +0x93 fp=0xa090f5c sp=0xa090f18 pc=0x80db513
    dlv_test.go:104: os.init()
    dlv_test.go:104: 	/usr/local/go/src/os/executable_procfs.go:30 +0x198 fp=0xa090f80 sp=0xa090f5c pc=0x80db728
    dlv_test.go:104: runtime.doInit(0x8162b00)
    dlv_test.go:104: 	/usr/local/go/src/runtime/proc.go:5625 +0x91 fp=0xa090f98 sp=0xa090f80 pc=0x80874c1
    dlv_test.go:104: runtime.doInit(0x8162800)
    dlv_test.go:104: 	/usr/local/go/src/runtime/proc.go:5620 +0x5b fp=0xa090fb0 sp=0xa090f98 pc=0x808748b
    dlv_test.go:104: runtime.doInit(0x8161540)
    dlv_test.go:104: 	/usr/local/go/src/runtime/proc.go:5620 +0x5b fp=0xa090fc8 sp=0xa090fb0 pc=0x808748b
    dlv_test.go:104: runtime.main()
    dlv_test.go:104: 	/usr/local/go/src/runtime/proc.go:191 +0x185 fp=0xa090ff0 sp=0xa090fc8 pc=0x807b095
    dlv_test.go:104: runtime.goexit()
    dlv_test.go:104: 	/usr/local/go/src/runtime/asm_386.s:1333 +0x1 fp=0xa090ff4 sp=0xa090ff0 pc=0x80a5581
    dlv_test.go:104: fatal error: runtime: stack split at bad time
```

https://travis-ci.com/github/go-delve/delve/jobs/372138035

--- a/.travis.yml
+++ b/.travis.yml
@@ -53,7 +53,7 @@ jobs:
   include:
     -  os: linux
        services: docker
-       env: go_32_version=1.14 # Linux/i386 tests held back to Go1.14 until issue #2134 is fixed
+       env: go_32_version=1.15.2 # Linux/i386 tests will fail on go1.15 prior to 1.15.2 (see issue #2134)
 
 script: >-
     if [ $TRAVIS_OS_NAME = "linux" ] && [ $go_32_version ]; then

--- a/_scripts/gen-travis.go
+++ b/_scripts/gen-travis.go
@@ -112,7 +112,7 @@ jobs:
   include:
     -  os: linux
        services: docker
-       env: go_32_version={{index .GoVersions 1}} # Linux/i386 tests held back to Go1.14 until issue #2134 is fixed
+       env: go_32_version={{index .GoVersions 0}}.2 # Linux/i386 tests will fail on go1.15 prior to 1.15.2 (see issue #2134)
 
 script: >-
     if [ $TRAVIS_OS_NAME = "linux" ] && [ $go_32_version ]; then

